### PR TITLE
Module for keeping track of docker containers

### PR DESCRIPTION
**Pull request by kevna**
This module gives a brief summary of docker containers that are currently running. Especially it allows a convenient reference for the IP on which a container is running.

It allows filtering based on the name or image tag of the containers.

### CODE REVIEW

File: py3status/modules/docker.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    containers = []
+    delimiter = " "
+    format = "{name} {image}"
```
Comment (lasers) [inline py3status/modules/docker.py:46]: - It should be changed to `format = "{format_container}"` where a list of containers will be given to `format`.
- Then a new config `format_container = "{name} {image}"` for the containers.
- Rename `delimiter` to `format_container_separator` to be used with config `format_container`. 
- - Take a look at module `nvidia_smi` or any modules containing config `format_*_separator` for some examples on looping / merging.

**Comment by ultrabug** [comment]
Thanks for your contribution @kevna !

Could you please address what lasers said? It's important we get the CI happy first :+1:

**Comment by ultrabug** [comment]
Please reopen if you feel like making your module up to py3status standards @kevna .

Thanks a lot again for your proposal!

**Comment by kevna** [comment]
@ultrabug apologies for dropping this, I moved to conky as I found I couldn't fit enough information in i3bar once I had several containers up.

If there's desire for the module I'll get the changes done, but I'm no longer using it myself.

**Comment by ultrabug** [comment]
No user base is no good so thanks for your time and for trying mate!

**Pull request closed without merging**